config/routes.rb

```diff
@@ -37,10 +37,8 @@
 
     resource :upload_supporting_evidence, only: :create
 
-    resource :sessions, only: :destroy do
-      get :authn_urls, on: :collection
-      get :multifactor, on: :member
-      get :identity_proof, on: :member
+    resource :sessions, only: [] do
+      get  :logout, to: 'sessions#logout'
```

Comment:
This is a GET? What about the redirect below that points to the resource with a DELETE verb?

Reply:
What? I don't follow. The FE right now is making a get request, it has to be a GET or else the front-end would break. The callback is POST, but IDME posts to that not the FE. There is no need for a delete.

Reply:
Fixed it. It did work locally as is, but removed it anyway because its confusing.